lib/Dancer2/Manual.pod

```diff
@@ -1219,9 +1204,8 @@ L<Template> module.
 In order to render a view, just call the
 L<template|Dancer2::Manual/template> keyword at the end of the action by
 giving the view name and the HASHREF of tokens to interpolate in the view
-(note that for convenience, the request, session, params, and vars are
-automatically accessible in the view, named C<request>, C<session>,
-C<params> and C<vars>) - for example:
+(note that for convenience, the request, session, route_parameters, body_parameters, query_parameters, and vars are
+automatically accessible in the view, named C<request>, C<session>, C<params>, and C<vars>) - for example:
```

Comment:
Is this the same text just stretched beyond 80 cols?

Also, `params` appears there too. Another miss.